docs/agent/config/v2.mdx

```diff
@@ -621,4 +619,237 @@ Edges automatically create an `edge` label, for example: `edge=edghts_2gYaK9XAVa
 
 :::
 
-<AgentConfigurations />
+## Agent Configuration
```

Comment:
I _could_ try to get clever and piece out just the parts that are shared (now that v2 and v3 have diverged), but I instead went the dumb and simple route since:

- It's easier to change without causing confusion with the v2 fields
- We're going to update these sections further, which makes anything that is "easy to throw away / delete" a good choice in my mind.
- It's grug brain and simple.